experiment/classifier.py

```diff
@@ -111,7 +153,7 @@ def main():
                 min(global_step / warmup_steps, 1 - (global_step - warmup_steps) / train_steps)
             ) # slanted triangular lr
         )
-    estimator = BaselineEstimator( # TODO: change to your estimator!
+    estimator = Naive_Label_Estimator( # TODO: change to your estimator!
```

Comment:
Is this the same as BaselineEstimator? Feel free to keep it if it contains new features.